docker/join.bash

```diff
@@ -1,11 +1,43 @@
 #!/usr/bin/env bash
 #
-# Typical usage: ./join.bash vrx
+# Typical usage: ./join.bash dave_nvidia
 #
 
 IMG=$(basename $1)
```

Comment:
In the above approach this line could be replaced with:
```
CONTAINER_ID=$1
```
And the rest of the logic can be removed.

Reply:
@bsb808 Sure. It looks like you might have also accidentally added the emacs backup file `Docker~`. I'll remove it (unless I'm misreading the commit).

Reply:
Thank you @M1chaelM 
I'm going to go ahead and merge.